public/features.jade

```diff
@@ -54,7 +54,7 @@
     h3.text-headline.text-uppercase  Flexible Development
     p.text-body.
       The choice of language is up to you. In addition to providing full support for ES5 and
-      TypeScript, and Dart, Angular 2 works equally well with ES6, Dart, and other languages that
+      TypeScript, Angular 2 works equally well with ES6, Dart, and other languages that
```

Comment:
Good catch! I'd rather solve it the other way:

In addition to providing full support for ES5, TypeScript, and Dart, Angular 2 works equally well with ES6 and other languages that